app/controllers/teachers_controller.rb

```diff
@@ -0,0 +1,67 @@
+class TeachersController < ApplicationController
+  before_action :authenticate_user!, :set_user
+
+  def show
+    @groups = @user.groups.all
+    @total_of_students = @user.group_members.count
+    # TODO: Add days of the weey to this query
+    @in_progress_group = @user.groups
+                              .where("from_time <= ?", Time.new)
+                              .where("to_time >= ?", Time.new).last
+  end
+
+  def show_students
```

Comment:
I don't really love this idea, why we don't have a students controller that depends on teachers? so we just keep the basic actions for each model

Reply:
fixed, create a new file `students_controller` inside `teachers` folder in controller namespace